### ISSUE TITLE
Face ID integration?

### ISSUE THREAD

**Issue by jdtaormi** [MiniKeePass/MiniKeePass#574]
The app looks great on iPhone X but FaceID isn’t available.

**Comment by sylido** [comment]
Yeah that would be really nice to have as there is no fingerprint reader anymore.

**Comment by shadowhywind** [comment]
App's that supported touchID before automatically support faceID. I have it enabled and it's working for me. In the MiniKeePass settings you do have to enable Touch ID. So some verbiage needs to be updated, but other then that, it appears to be already working.

**Comment by jdtaormi** [comment]
Oh awesome. Thank you for letting me know!

Justin Taormina
jdtaormi@gmail.com

**Comment by sylido** [comment]
@shadowhywind that’s not neccessarily the case, my bank apps don’t work - there was a message saying they would support it soon. So I had the touchid enabled in the minikeepass settings and it didn’t and still doesn’t work. Maybe I’m missing something ?

**Comment by jdtaormi** [comment]
I had to disable touchID, then reenable touchID, then restart the app and a popup asked “Do you want to use FaceID?”

Justin Taormina
jdtaormi@gmail.com

**Comment by sylido** [comment]
@jdtaormi Hmm, thanks, seems that the touchid is greyed out for me and I can't disable/enable it. I'll try some stuff see if I can get it to ungrey itself.

EDIT: I guess I had to enter my database to unlock it, then disable the already enabled touchID, by adding a PIN. Then had to close the app and open it again, after entering the PIN it asked me about faceID. Also had to let iCloud's keychain to store my db password again.

**Comment by jamisonbennett** [comment]
I have posted a pull request which is under review here https://github.com/MiniKeePass/MiniKeePass/pull/639.

There was a question above about "the touchid is greyed out for me and I can't disable/enable it". This is likely because the touch ID/face ID is not enabled unless the PIN is enabled and a finger/face is enrolled in touch ID/face ID.

**Comment by jamisonbennett** [comment]
The face ID for the app may also be disabled. https://www.macobserver.com/tips/quick-tip/disable-face-id-for-specific-apps/ has instructions for how that setting may be changed.
